src/plugins/intel_cpu/src/node.h

```diff
@@ -192,11 +204,12 @@ class Node {
         return childEdges;
     }
 
-    const EdgePtr getParentEdgeAt(size_t idx) const;
-    virtual const EdgePtr getChildEdgeAt(size_t idx) const;
+    EdgePtr getParentEdgeAt(size_t idx) const;
+    EdgePtr getChildEdgeAt(size_t idx) const;
 
-    const std::vector<EdgePtr> getParentEdgesAtPort(size_t idx) const;
-    const std::vector<EdgePtr> getChildEdgesAtPort(size_t idx) const;
+    EdgePtr getParentEdgeAtPort(size_t idx) const;
+    std::vector<EdgePtr> getChildEdgesAtPort(size_t idx) const;
+    EdgePtr getAnyChildEdgeAtPort(size_t idx) const;
```

Comment:
I do not really like the idea of having this method in general, since it does not really express the potential usage.
In 99% of the time, it is used to access the Memory at port (the main optimization target)
One of the alternatives could be introducing two new public methods instead, **getSrcMemoryAtPort** and **getDstMemoryAtPort** and making **getAnyChildEdgeAtPort** a local lambda of the **getDstMemoryAtPort**.
@dmitry-gorokhov @maxnick, what do you think?